### DISCUSSION TITLE
安装好docker后打不开地址，端口已放行

### DISCUSSION THREAD

**Discussion by ZBSSA** [chanify/chanify#100] [Q&A]
centos7.6的系统，用的宝塔的docker
![_20211722061725](https://user-images.githubusercontent.com/43901213/134326365-e22a9600-8e57-4d95-86d1-27c21a1f85f2.png)

**Comment by wizjin** [comment]
你好，你这里 Chanify 使用的端口是 80，而对外开放的端口是 5800，这个是不通的。

可以在 docker 命令上添加端口映射的参数试一下

```bash
docker run -it -p 0.0.0.0:5800:80
```

**Reply by ZBSSA** [reply]
![_20214422064411](https://user-images.githubusercontent.com/43901213/134329951-1d41eb6d-dd97-4999-9b87-3fc83920cb2f.png)
这样显示，没效果呀

**Comment by wizjin** [comment]
请把 `-p 0.0.0.0:5800:80` 加在原来的命令中

```bash
docker run -it -p 0.0.0.0:5800:80 -v /my/data:/root/.chanify wizjin/chanify:latest serve --name=mynode --endpoint=http://xxxx:5800
```

**Reply by ZBSSA** [reply]
按照您的建议使用上述方式安装后还是无法使用，映射，开放端口，和docker日志如上

**Comment by wizjin** [comment]
@ZBSSA 你好，我刚才试了一下，用浏览器能打开 `http://180.76.53.167:5800` 能打开网页，然后用 Chanify 扫描网页上的二维码能显示节点添加页面。

请问无法使用具体是指什么样的情况？能详细描述一下吗？

谢谢了

**Reply by ZBSSA** [reply]
我的电脑和手机都打不开网页，不知道为什么呀，就是这样显示，我试了三四台电脑都不行
![_20213423093428](https://user-images.githubusercontent.com/43901213/134516639-63e2271e-0ac7-45ac-8953-0043bbe7c550.png)

**Comment by wizjin** [comment]
@ZBSSA 我这里试了一下手机和PC都是可以的，可能是网络问题吧
![441632404970_ pic](https://user-images.githubusercontent.com/5716629/134519635-78acdefc-297d-4714-8c96-1c769dea5182.jpg)
<img width="834" alt="WeChat43c5080c07b9d951a5af993edebc02bb" src="https://user-images.githubusercontent.com/5716629/134520068-f94fda29-7054-418c-9ae3-9bd4e61b0152.png">

**Reply by ZBSSA** [reply]
好吧，那应该是我这里网络环境的问题，非常感谢

**Comment by wizjin** [comment]
@ZBSSA 建议你可以换一个端口再试一下，因为 5800 和 5900 是 vnc 远程桌面的端口，可能网络上有设备封这个端口的。

**Reply by ZBSSA** [reply]
好的，我试一下别的端口，感谢您的耐心解答

**Reply by wizjin** [reply]
不客气，有问题欢迎随时联系。
